src/containerapp/azext_containerapp/custom.py

```diff
@@ -4297,3 +4306,195 @@ def delete_workload_profile(cmd, resource_group_name, env_name, workload_profile
+            else:
+                for run_images_prop in run_images_props:
+                    if run_images_prop["name"].find("mcr.microsoft.com/oryx/builder") != -1:
+                        run_images_prop = run_images_prop["name"].split(":")
+                        run_images_tag = run_images_prop[1]
+                        # Based on Mariners
+                        if run_images_tag.find('mariner') != -1:
+                            check_result = patchable_check(run_images_tag, oryx_run_img_tags, inspect_result=inspect_result)
+                            results.append(check_result)
+                        else:
+                            results.append(dict(targetContainerName=inspect_result["targetContainerName"], targetContainerAppName=inspect_result["targetContainerAppName"], targetContainerAppEnvironmentName=inspect_result["targetContainerAppEnvironmentName"], targetResourceGroup=inspect_result["targetResourceGroup"], targetImageName=inspect_result["image_name"], oldRunImage=inspect_result["remote_info"]["run_images"]["name"], newRunImage=None, id=None, reason=failed_reason))
+                    else:
+                        # Not based on image from mcr.microsoft.com/dotnet
+                        results.append(dict(targetContainerAppName=inspect_result["targetContainerAppName"], targetContainerAppEnvironmentName=inspect_result["targetContainerAppEnvironmentName"], targetResourceGroup=inspect_result["targetResourceGroup"], oldRunImage=inspect_result["remote_info"]["run_images"], newRunImage=None, id=None, reason=mcr_check_reason))
+    # Make sure that we clear the first line before showing the new output
+    print("\r                                   \r", end="", flush=True)
+    if show_all is False:
+        results = [result for result in results if result["id"] is not None]
+    if not results:
+        print("No container apps available to patch at this time. Use --show-all to show the container apps that cannot be patched.")
+        return
+    return results
+
+
+def patch_get_image_inspection(pack_exec_path, img, info_list):
+    if (img["imageName"].find("run-dotnet") != -1) and (img["imageName"].find("cbl-mariner") != -1):
+        inspect_result = {"remote_info": {"run_images": [{"name": "mcr.microsoft.com/oryx/builder:" + img["imageName"].split(":")[-1]}]}, "image_name": img["imageName"], "targetContainerName": img["targetContainerName"], "targetContainerAppName": img["targetContainerAppName"], "targetContainerAppEnvironmentName": img["targetContainerAppEnvironmentName"], "targetResourceGroup": img["targetResourceGroup"]}
```

Comment:
@snehapar9 @harryli0108 -- nit: it would be nice to expand this out to its JSON form to make it easier to read

Reply:
Updated: changed the format here to match JSON format